etc/kayobe/inventory/overcloud

```diff
@@ -31,6 +27,11 @@ cpu-e-1026 ipmi_address=10.202.0.15 bmc_type=idrac
 cpu-e-1027 ipmi_address=10.202.0.16 bmc_type=idrac
 cpu-e-1028 ipmi_address=10.202.0.17 bmc_type=idrac
 
+# only put monitoring on ceph nodes
```

Comment:
I suppose an alternative would be to add the ceph group to the relevant services in `overcloud-components.j2` and then the cinder components could be left alone?